src/action-ros-ci.ts

```diff
@@ -111,6 +111,17 @@ export async function execBashCommand(
 	});
 }
 
+//list of valid ROS distributions
+const validDistro: string[] = [
+	"kinetic",
```

Comment:
we should be checking valid ros1 and ros2 distros separately, to validate the distro argument matches the version it is targeting. It should be an error if I specify "kinetic" for target-ros2-distro